app/src/main/java/io/mdevlab/ocatraining/util/Helper.java

```diff
@@ -27,36 +25,6 @@
 
 
     /**
-     * @param strings Set of string objects
```

Comment:
Unused code, has nothing to do with the topic of this PR, just noticed it wasn't being used and removed it